docs/DeveloperGuide.md

```diff
@@ -592,38 +606,38 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
By the convention for markdown they gave, should leave as 1. for these pointer. Same for all the other ones.

Reply:
Done